.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Build
+        run: yarn run build
+      - uses: actions/upload-artifact@v3
+        name: Upload Artifacts
+        with:
+          name: tauri-web
+          path: dist/*
+  build-linux:
+    runs-on: ubuntu-latest
```

Comment:
Ideally, we want the oldest supported Ubuntu LTS for widest compatibility of the generated executable

Reply:
I will change this